core/imagefs_linux.go

```diff
@@ -63,6 +63,15 @@ func (ds *dockerService) imageFsInfo() (*runtimeapi.ImageFsInfoResponse, error)
 	}
 	logrus.Debugf("Total used bytes by docker images: %v", totalImageSize)
 
+	var totalRWLayerSize uint64
+	for cid, _ := range ds.containerStatsCache.stats {
```

Comment:
There's a RW lock for accessing `containerStatsCache`, maybe a better way is to add a new method for `containerStatsCache` in `core/stats.go`?